cookbooks/bcpc/templates/default/apache-graphite-web.conf.erb

```diff
@@ -5,16 +5,16 @@
 ################################################
 
 Listen <%="#{node[:bcpc][:graphite][:ip]}"%>:<%="#{node[:bcpc][:graphite][:web_port]}"%>
```

Comment:
graphite should be listening to the IP already here right?

Can you pasted the rendered template for a VM cluster?

Reply:
After this PR is applied, the change is as follow:
```
  * template[/etc/apache2/sites-available/graphite-web.conf] action create
    - update content in file /etc/apache2/sites-available/graphite-web.conf from aa9603 to ba4387
    --- /etc/apache2/sites-available/graphite-web.conf  2017-12-27 15:40:58.585553693 -0500
    +++ /etc/apache2/sites-available/.chef-graphite-web20171228-9983-7titp4.conf        2017-12-28 13:44:57.447231632 -0500
    @@ -5,10 +5,11 @@
     ################################################
     
     Listen 10.0.100.5:8888
    +Listen 10.0.100.12:8888
     NameVirtualHost 10.0.100.12:443
     
     
    -<VirtualHost 10.0.100.5:8888 10.0.100.5:443  >
    +<VirtualHost 10.0.100.5:8888 10.0.100.12:443 10.0.100.12:8888>
     
         ServerName "graphite.bcpc.example.com"
         DocumentRoot "/opt/graphite/webapp"
```

Reply:
from chef-shell:
```
chef (12.19.36)> pp node['bcpc']['graphite']['ip']
"10.0.100.5"
 => "10.0.100.5" 
chef (12.19.36)> pp node['bcpc']['management']['ip']
"10.0.100.11"
 => "10.0.100.11" 
chef (12.19.36)> pp node['bcpc']['management']['vip']
"10.0.100.5"
 => "10.0.100.5" 
```